agents/lakeshore372/LS372_agent.py

```diff
@@ -314,17 +326,36 @@ def start_acq(self, session, params=None):
                     # For data feed
                     data['data'][channel_str + '_T'] = temp_reading
                     data['data'][channel_str + '_R'] = res_reading
+                    session.app.publish_to_feed('temperatures', data)
+                    self.log.debug("{data}", data=session.data)
 
                     # For session.data
                     field_dict = {channel_str: {"T": temp_reading,
                                                 "R": res_reading,
                                                 "timestamp": current_time}}
-
                     session.data['fields'].update(field_dict)
 
-                session.app.publish_to_feed('temperatures', data)
-
-                self.log.debug("{data}", data=session.data)
+                    # Also queries control channel if enabled
+                    if self.control_chan_enabled:
+                        temp = self.module.get_temp(unit='kelvin', chan=0)
+                        res = self.module.get_temp(unit='ohms', chan=0)
+                        cur_time = time.time()
+                        data = {
+                            'timestamp': time.time(),
+                            'block_name': 'control_chan',
+                            'data': {
+                                'control_T': temp,
+                                'control_R': res
+                            }
+                        }
+                        session.app.publish_to_feed('temperatures', data)
+                        self.log.debug("{data}", data=session.data)
+                        # Updates session data w/ control field
+                        session.data['fields'].update({
+                            'control': {
+                                'T': temp, 'R': res, 'timestamp': cur_time
```

Comment:
Please add an example line to the `session.data` example in the docstring with "control" as a key?